pkg/uroot/uroot.go

```diff
@@ -177,7 +177,7 @@ func CreateInitramfs(opts Opts) error {
 		} else {
 			// plain old syntax
 			src = file
-			dst = file
+			dst = strings.TrimLeft(file, "/")
```

Comment:
ok, I did this differently.First off, it fails with paths like ///bin/bash. So.
src = filepath.Clean(src)
dst = src
if path.IsAbs(dst) {
dst = filepath.Rel("/", dst)
}

or some such. As is it's a bit fragile and people can break it if their scripts don't give you clean paths.

Reply:
I've addressed the comments above. Checking on this one shortly